template.yaml

```diff
@@ -80,6 +80,11 @@ Parameters:
     Description: The number of days to retain the Cloudwatch Logs of the lambda.
     Default: "1"
 
+  MaxDescribeScalingActivitiesPages:
+    Type: Number
+    Description: The number of pages to retrive for DescribeScalingActivity. Negative numbers mean unlimited.
+    Default: "-1"
```

Comment:
should we default this to something low-but-reasonable? if the goal of this work is to consume AWS rate limits less, we should probably make the default something that will reduce the rate-limit-stress we're putting on customers

Reply:
Maybe, I believe the fact that we were searching for non-existent scale-in events is major cause of the rate limiting. We can also tweak this parameter from the elastic ci stack, so I propose we leave it unconstrained here and pass in a limit through the elastic stack if we need to.